nemo/collections/asr/modules/conformer_encoder.py

```diff
@@ -118,6 +118,14 @@ class ConformerEncoder(NeuralModule, StreamingEncoder, Exportable, AccessMixin):
             All layers before this will never be dropped. Note that drop
             probability will be adjusted accordingly if mode is "linear" when
             start layer is > 1. Defaults to 1.
+        global_tokens (int): number of tokens to be used for global attention.
+            Only relevant if self_attention_model is 'rel_pos_local_attn'.
+            Defaults to 0.
+        global_tokens_placing (str): where the tokens should be placed.
```

Comment:
What other values are valid for this parameter?

Reply:
Replaced this with a different parameter for global tokens spacing